lang/en/local_edit_course_notification.php

```diff
@@ -26,7 +26,11 @@
 defined('MOODLE_INTERNAL') || die();
 
 $string['notificationstatement'] = '';
-$string['notificationtype'] = 'Information';
+$string['notificationtype'] = \core\output\notification::NOTIFY_INFO;
```

Comment:
It doesn't make sense to have it as a lang string. Can you please remove this line and move \core\output\notification::NOTIFY_INFO to setting.php to be a default setting.

Reply:
I understand what you mean.
We don't need to use lang file if default values are for select, or empty.
I moved default value to settings.php